### PR TITLE
Add elixir file with violations

### PR DESCRIPTION
**Pull request by jakecraige**

### CODE REVIEW

File: test.ex [hunk 1]
```diff
@@ -0,0 +1,7 @@
+if !true do
+  IO.puts "Hello"
+else
+  # TODO: make this do something duh
+  # FIXME: more errors
```
Comment (jakecraige) [inline test.ex]: Found a FIXME tag in a comment: # FIXME: more errors

File: test.ex [hunk 2]
```diff
@@ -0,0 +1,7 @@
+if !true do
+  IO.puts "Hello"
+else
+  # TODO: make this do something duh
```
Comment (jakecraige) [inline test.ex]: Found a TODO tag in a comment: # TODO: make this do something duh

File: test.ex [hunk 3]
```diff
@@ -0,0 +1,7 @@
+if !true do
```
Comment (jakecraige) [inline test.ex]: Avoid negated conditions in if-else blocks.

File: test.ex [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  IO.puts "Hello"
+else
+  # TODO: make this do something duh
```
Comment (jakecraige) [inline test.ex]: Found a TODO tag in a comment: # TODO: make this do something duh

File: test.ex (hunk 3 above)
Comment (houndci-staging) [inline test.ex]: Avoid negated conditions in if-else blocks.

File: test.ex [hunk 5]
```diff
@@ -0,0 +1,7 @@
+if !true do
+  IO.puts "Hello"
+  # FIXME: something something
+else
+  # TODO: make this do something duh
```
Comment (houndci-staging) [inline test.ex]: Found a TODO tag in a comment: # TODO: make this do something duh

File: test.ex [hunk 6]
```diff
@@ -0,0 +1,7 @@
+if !true do
+  IO.puts "Hello"
+  # FIXME: something something
```
Comment (houndci-staging) [inline test.ex]: Found a FIXME tag in a comment: # FIXME: something something

File: test.ex [hunk 7]
```diff
@@ -0,0 +1,6 @@
+if !true do
```
Comment (houndci-staging) [inline test.ex]: Avoid negated conditions in if-else blocks.

File: test.ex (hunk 4 above)
Comment (houndci-staging) [inline test.ex]: Found a TODO tag in a comment: # TODO: make this do something duh

**Pull request merged**